### PR TITLE
Allow for multiple results in the plotGui script.

### PR DESCRIPTION
**Pull request by legordian**
Make plotting of several fit results over each other possible in the plotGui script and clean it up a little bit.

### CODE REVIEW

File: src/rootscripts/plotGui.C [hunk 1]
```diff
@@ -318,150 +337,199 @@ void plotGuiMainFrame::PrintSelected(const int sel1, const int sel2)
...
+			gIm->SetPointError(i, _binWidth*0.5, sqrt(rhoCov[1][1]));
+		}// end loop over bins
+
+	intensity1Graph->Add(g1);
+	intensity2Graph->Add(g2);
+	phaseGraph->Add(gph);
+	phaseGraph->Add(gphP1);
+	phaseGraph->Add(gphM1);
+	reGraph->Add(gRe);
+	imGraph->Add(gIm);
```
Comment (suhlatwork) [inline src/rootscripts/plotGui.C]: Are these whitespace errors, or is this a problem with the display in a web browser?

File: src/rootscripts/plotGui.C [hunk 2]
```diff
@@ -318,150 +337,199 @@ void plotGuiMainFrame::PrintSelected(const int sel1, const int sel2)
...
+			}
 
-		gph->SetPoint(i, result->massBinCenter()*0.001, ph);
-		gph->SetPointError(i, _binWidth*0.5, phErr);
+			g2->SetPoint(i, result->massBinCenter()*0.001, intensity2);
+			g2->SetPointError(i, _binWidth*0.5, intensityErr2);
+
+			// check if we should make a transformation by 2pi
+			// this is needed because of cyclical variable phi
+			if(i>11){
```
Comment (suhlatwork) [inline src/rootscripts/plotGui.C]: This is already been in the previous version, but why is it there? Are the fits in the lower massbins that unstable?

Comment (legordian) [inline src/rootscripts/plotGui.C]: No idea why that is there. From the looks of it, it probably is one of the first attempts to treat jumping phases. As the phases are now plotted +- 360 degrees anyway, I removed this.

File: src/rootscripts/plotGui.C (hunk 1 above)
Comment (legordian) [inline src/rootscripts/plotGui.C]: These were whitespace errors, fixed now.

**Comment by legordian** [comment]
I implemented your suggestions (see line notes). Thank you very much!

**Pull request merged**